doc/source/ray-core/monitoring-debugging/monitoring-debugging.rst

```diff
@@ -0,0 +1,17 @@
+Monitoring and Debugging
```

Comment:
nit: Monitoring and Troubleshooting?

Reply:
This part was inspired by https://kubernetes.io/docs/tasks/debug/

I have no preference, so lmk if you strongly feel about this title.

Reply:
No strong preference either, just feel a bit more consistent since the subpages are troubleshooting-*. But I guess people just use these two words interchangeably.